### ISSUE TITLE
Comments on the specification

### ISSUE THREAD

**Issue by gdude2002** [samis/OC-Port-Registry#1]
- We'll need to have an actual webapp - So let's try Cyclone. I see you've copied one of the examples in.
- We'll need to have some kind of data storage. It doesn't really get better than MongoDB for apps like this, so let's use that and PyMongoDB.
- Auth0 isn't viable - we would need to pay for it. Instead, we have a few options.
  - GitHub logins
  - Cyclone's built-in Twitter and Facebook login handlers
- As for the design of the site itself, I suggest looking at Semantic-UI.

**Comment by samis** [comment]
Fair Enough. I did notice the default Twitter / Facebook items. GitHub auth seems especially viable for OC (though I cannot be 100% accurate on this due to not frequenting the OC forums.) MongoDB does look interesting, but I've never done NoSQL before so this might be a nice introduction :)
